deploy/kube/mixer.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: mixer
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: mixer
+      annotations:
+        prometheus_port: "42422"
```

Comment:
i think this is using black-magic to get prom to scrape at the right addr:port.

we don't have a good way of ensuring the .yaml matches the mixer adapter configs, etc., yet., so i think this is a reasonable first step.  perhaps we should comment on it here, however?